base/settings/develop.py

```diff
@@ -14,6 +14,6 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, '../../db.sqlite3'),
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
```

Comment:
also, for further functionalities you will need to use postgresql and postgis

Reply:
Is it okay if I deal with it at that point or do you want me to do it for this task?